---
layout: page
title: Publications
subtitle: of the priority research area Evolutionary Systems Biology
use-site-title: true
---
<p align='center'>
	<img src="img/esb-logo.png" alt='ESB logo' height="200px">
</p>

## Bäurle
Liu H-c, Lämke J, Lin S-y, Hung M-J, Liu, Kuan-Ming, Charng Y-y, Bäurle I (2018) Distinct heat shock factors and chromatin modifications mediate the organ-autonomous transcriptional memory of heat stress. Plant Journal 95, 401-413. DOI: 10.1111/tpj.13958.
 
Lämke J, Bäurle I (2017). Epigenetic and chromatin-based mechanisms in environmental stress adaptation and stress memory in plants. Genome Biology 18, 124. Doi: 10.1186/s13059-017-1263-6.
 
Brzezinka K*, Altmann S*, Czesnick H, Nicolas P, Górka M, Benke E, Kabelitz T, Jähne F, Graf A, Kappel C, Bäurle I (2016). Arabidopsis FORGETTER1 mediates stress-induced chromatin memory through nucleosome remodeling. eLife 5, e17061. Doi: 10.7554/eLife.17061. * denotes equal contribution of authors.
 
Lämke J, Brzezinka K, Altmann S, Bäurle I (2016). A hit-and-run heat shock factor governs sustained histone methylation and transcriptional stress memory. EMBO J. 35, 162-175.
 
Kabelitz T, Kappel C, Henneberger K, Benke E, Nöh C, Bäurle I (2014). eQTL mapping of transposon silencing reveals a position-dependent stable escape from epigenetic silencing and transposition of AtMu1 in the Arabidopsis thaliana lineage. Plant Cell 26, 3261-71.

## Dittmann
Hackenberg, C., Hakanpää, J., Cai, F., Antonyuk, S., Eigner, S., Meissner, S., Laitaoja, M., Jänis, J., Kerfeld, C.A., Dittmann, E., Lamzin, V.S. (2018) Structural and functional insights into the unique CBS-CP12 fusion protein family in cyanobacteria. Proc. Natl. Acad. Sci., 115(27):7141-7146

Reyna-González E, Schmid B, Petras D, Süssmuth RD, Dittmann E (2016) Leader-peptide free in vitro reconstitution of microviridin biosynthesis enables the design of synthetic protease targeted libraries. Angew. Chem. Int. Ed. 55: 9398-9401 

Meyer S, Kehr JC, Mainz A, Dehm D, Petras D, Süssmuth RD, Dittmann E (2016) Biochemical dissection of the natural diversification of microcystin provides lessons for synthetic biology of NRPS. Cell Chem. Biol. 23: 462-471

Liaimer A, Helfrich EH, Hinrichs K, Guljamow A, Ishida K, Hertweck C and Dittmann E (2015) Nostopeptolide plays a governing role in cellular differentiation of the symbiotic cyanobacterium Nostoc punctiforme. Proc. Natl. Acad. Sci. 112: 1862-1867

Meissner S, Steinhauser D and Dittmann E (2015). Metabolomic analysis indicates a pivotal role of the hepatotoxin microcystin in high light adaptation of Microcystis. Environ. Microbiol. 17: 1497-1509

## Gräf
Pitzen, V., Askarzada, S., Gräf, R., Meyer, I. (2018) CDK5RAP2 is an essential scaffolding protein of the corona of the Dictyostelium centrosome. Cells, 7, 04, doi:10.3390/cells7040032

Meyer, I., Peter, T., Batsios, P., Kuhnert, O., Krüger-Genge, A., Camurça, C. and Gräf, R. (2017) CP39, CP75 and CP91 are major structural components of the Dictyostelium centrosome's core structure. Eur. J. Cell Biol., 96, 119-130

Putzler, S., Meyer, I., Gräf, R. (2016) CP91 is a component of the Dictyostelium centrosome involved in centrosome biogenesis. Eur. J. Cell Biol., 95, 124-135.

Batsios, P., Meyer, I., Gräf, R. (2016) Proximity-dependent biotin identification (BioID) in Dictyostelium amoebae. Methods Enzymol. 569, 23-43.

Gräf, R., Batsios, P., and Meyer, I. (2015) Evolution of centrosomes and the nuclear lamina: Amoebozoan assets. Eur. J. Cell Biol., 94, 249-256. 

## Grebe
Nakamura M, Claes AR, Grebe T, Hermkes R, Viotti C, Ikeda Y, Grebe M (2018). Auxin and ROP GTPase signaling of polar nuclear migration in root epidermal hair cells. Plant Physiology 176: 378-391.
 
Mao H, Aryal B, Langenecker T, Hagmann J, Geisler M, Grebe M (2017). Arabidopsis BTB/POZ protein-dependent PENETRATION3 trafficking and disease susceptibility. Nature Plants 3: 854-858.
 
Mao H, Nakamura M, Viotti C, Grebe M (2016). A framework for lateral membrane trafficking and polar tethering of the PEN3 ATP-binding cassette transporter. Plant Physiology: 2245-2260.
 
Stanislas T, Hüser A, Barbosa IC, Kiefer CS, Brackmann K, Pietra S, Gustavsson A, Zourelidou M, Schwechheimer C, Grebe M (2015). Arabidopsis D6PK is a lipid domain-dependent mediator of root epidermal planar polarity. Nature Plants 1: 15162.
 
Kiefer CS, Claes AR, Nzayisenga JC, Pietra S, Stanislas T, Hüser A, Ikeda Y, Grebe M (2015). Arabidopsis AIP1-2 restricted by WER-mediated patterning modulates planar polarity. Development 142:151-161.

## Hartmann
A Barlow, JA Cahill, S Hartmann, C Theunert, G Xenikoudakis, GG Fortes, JLA Paijmans, G Rabeder, C Frischauf, A Grandal-dAnglade, A Garcia-Vazquez, M Murtskhvaladze, U Saarma, P Anijalg, T Skrbinsek, G Bertorelle, B Gasparian, G Bar-Oz, R Pinhasi, M Slatkin, L Dalen, B Shapiro, and M Hofreiter. (2018) Partial genomic survival of cave bears in living brown bears. Nature Ecology & Evolution, 2(10):1563-1570.

L Hilgers, S Hartmann, M Hofreiter, T von Rintelen. (2018) Novel genes, ancient genes, and gene co-option contributed to the genetic basis of the radula, a molluscan innovation. Molecular Biology and Evolution, 35(7):1638-1652.

M Westbury, S Baleka, A Barlow, S Hartmann, JLA Paijmans, A Kramarz, AM Forasiepi, M Bond, JN Gelfo, MA Reguero, P Lopez-Mendoza, M Taglioretti, F Scaglia, A Rinderknecht, W Jones, F Mena, G Billet, C de Muizon, JL Aguilar, RDE MacPhee, M Hofreiter (2017) A mitogenomic timetree for Darwins enigmatic South American mammal Macrauchenia patachonica. Nature Communications, 8:15951.

K Wecek, S Hartmann, JLA Paijmans, U Taron, G Xenikoudakis, JA Cahill, PD Heintzman, B Shapiro, G Baryshnikov, AN Bunevich, JJ Crees, R Dobosz, N Manaserian, H Okarma, M Tokarska, ST Turvey, JM Wojcik, W Zyla, JM Szymura, M Hofreiter, A Barlow. (2017) Complex admixture preceded and followed the extinction of wisent in the wild. Molecular Biology and Evolution 34(3):598612

S Hartmann, N Hasenkamp, J Mayer, J Michaux, S Morand, CJ Mazzoni, AL Roca, AD Greenwood. (2015) Endogenous murine leukemia retroviral variation across wild European and inbred strains of house mouse. BMC Genomics. 16:613

## Hofreiter
Barlow, A., Cahill, J.A., Hartmann, S., Theunert, C., Xenikoudakis, G., Fortes, G.G., Paijmans, J.L.A., Rabeder, G., Frischauf, C., Grandal-d'Anglade, A., García-Vázquez, A., Murtskhvaladze, M., Saarma, U., Anijalg, P., Skrbinšek, T., Bertorelle, G., Gasparian, B., Bar-Oz, G., Pinhasi, R., Slatkin, M., Dalén, L., Shapiro, B. and Hofreiter, M. (2018) Partial genomic survival of cave bears in living brown bears. Nature Ecology and Evolution, 2(10), 1563-1570. 

Palkopoulou, E., Lipson, M., Mallick, S., Nielsen, S., Rohland, N., Baleka, S., Karpinski, E., Ivancevic, A.M., To, T.H., Kortschak, R.D., Raison, J.M., Qu, Z., Chin, T.J., Alt, K.W., Claesson, S., Dalén, L., MacPhee, R.D.E., Meller, H., Roca, A.L., Ryder, O.A., Heiman, D., Young, S., Breen, M., Williams, C., Aken, B.L., Ruffier, M., Karlsson, E., Johnson, J., Di Palma, F., Alfoldi, J., Adelson, D.L., Mailund, T., Munch, K., Lindblad-Toh, K., Hofreiter, M., Poinar, H. and Reich, D. (2018) A comprehensive genomic history of extinct and living elephants. Proc Natl Acad Sci USA 115, E2566-E2574 

Gaunitz, C., Fages, A., Hanghøj, K., Albrechtsen, A., Khan, N., Schubert, M., Seguin-Orlando, A., Owens, I.J., Felkel, S., Bignon-Lau, O., de Barros Damgaard, P., Mittnik, A., Mohaseb, A.F., Davoudi, H., Alquraishi, S., Alfarhan, A.H., Al-Rasheid, K.A.S., Crubézy, E., Benecke, N., Olsen, S., Brown, D., Anthony, D., Massy, K., Pitulko, V., Kasparov, A., Brem, G., Hofreiter, M., Mukhtarova, G., Baimukhanov, N., Lõugas, L., Onar, V., Stockhammer, P.W., Krause, J., Boldgiv, B., Undrakhbold, S., Erdenebaatar, D., Lepetz, S., Mashkour, M., Ludwig, A., Wallner, B., Merz, V., Merz, I., Zaibert, V., Willerslev, E., Librado, P., Outram, A.K. and Orlando, L. (2018) Ancient genomes revisit the ancestry of domestic and Przewalski's horses. Science, 360, 111-114.

Westbury, M. Baleka, S., Barlow, A., Hartmann, S., Paijmans, J.L.A., Kramarz, A., Forasiepi, A.M., Bond, M., Gelfo, J.N., Reguero, M.A., Mendoza, P.L., Taglioretti, M., Scaglia, F., Rinderknecht, A., Jones, W., Mena, F., Billet, G., de Muizon, C., Aguilar, J.L., MacPhee, R.D.E. and Hofreiter, M. (2017) A mitogenomic timetree for Darwin’s enigmatic South American mammal Macrauchenia patachonica. Nature Communications, 8, 15951. 

Welcker, F., Collins, M.J., Thomas, J.A., Wadsley, M., Brace, B., Cappellini, E., Turvey, S.T., Reguero, M., Gelfo, J.N., Kramarz, A.,  Burger, J., Thomas-Oates, J., Ashford, D.A., Ashton, P., Rowsell, K., Porter, D.M., Kessler, B., Fisher, R., Baessmann, C., Kaspar, S., Olsen, J., Kiley, P., Elliot, J., Kelstrup, C., Mullin, V., Hofreiter, M., Willerslev, E., Hublin, J.-J., Orlando, L., Barnes I. & MacPhee, R.D.E. (2015)  Ancient proteins resolve the evolutionary history of Darwin's South American ungulates. Nature, 522, 81-84.

## Lenhard
Fujikura, U., Jing, R., Hanada, A., Takebayashi, Y., Sakakibara, H., Yamaguchi, S., Kappel, C., Lenhard, M. (2018). Variation in splicing efficiency underlies morphological evolution in Capsella. Developmental Cell, doi: 10.1016/j.devcel.2017.11.022

Sicard, A., Kappel, C., Lee, Y.-W., Wozniak, N., Marona, C., Stinchcombe, J.R., Wright, S.I., Lenhard, M. (2016). Standing genetic variation in a tissue-specific enhancer underlies selfing-syndrome evolution in Capsella. Proc Natl Acad Sci USA; doi: 10.1073/pnas.1613394113

Sas, C., Müller, F., Kappel, C., Kent, T.V., Wright, S.I., Hilker, M., Lenhard, M. (2016). Repeated inactivation of the first committed enzyme underlies the loss of benzaldehyde emission after the selfing transition in Capsella. Current Biology; doi: 10.1016/j.cub.2016.10.026

Huu, C.N., Kappel, C., Keller, B., Sicard, A., Takebayashi, Y., Breuninger, H., Nowak, M.D., Bäurle, I., Himmelbach, A., Burkart, M., Ebbing-Lohaus, T., Sakakibara, H., Altschmied, L., Conti, E., Lenhard, M. (2016). Presence versus absence of CYP734A50 underlies the style-length dimorphism in primroses. eLIFE, doi: 10.7554/eLife.17956

Sicard, A., Kappel, C., Josephs, E.B., Lee, Y.W., Marona, C., Stinchcombe, J.R., Wright, S.I., Lenhard, M. (2015). Divergent sorting of a balanced ancestral polymorphism underlies the establishment of gene-flow barriers in Capsella. Nature Communications, doi: 10.1038/ncomms8960

## Müller-Röber
Hochrein, L., Mitchell, L.A., Schulz, K., Messerschmidt, K., and Mueller-Roeber, B. (2018) L-SCRaMbLE as a tool for light-controlled Cre-mediated recombination in yeast. Nat Commun. 9(1):1931. doi: 10.1038/s41467-017-02208-6.

Hochrein, L., Machens, F., Gremmels, J., Schulz, K., Messerschmidt, K., and Mueller-Roeber, B. (2017) AssemblX: A user friendly toolkit for rapid and reliable multi-gene assemblies. Nucl. Acids Res. 6(9):1742-1756.

Sedaghatmehr, M., Mueller-Roeber, B. and Balazadeh, S. (2016) The plastid metalloprotease FtsH6 and small heat shock protein HSP21 jointly regulate thermomemory in Arabidopsis. Nature Commun., 7: 12439.

Proost, S., Van Bel, M., Vaneechoutte, D., Van de Peer, Y., Inzé, D., Mueller-Roeber, B., and Vandepoele K. (2015) PLAZA 3.0: an access point for plant comparative genomics. Nucl. Acids Res., 43 (Database issue), D974-981.

Lu, D., Wang, T., Persson, S., Mueller-Roeber, B., Schippers, J.H. (2014) Transcriptional control of ROS homeostasis by KUODA1 regulates cell expansion during leaf development. Nat. Commun. 5: 3767.

## Nikoloski
Küken A, Sommer F, Yaneva-Roder L, Mackinder CML, Höhne M, Geimer S, Jonikas MC, Schroda M, Stitt M, Nikoloski Z, Mettler-Altmann T (2018) Effects of microcompartmentation on flux distribution and metabolic pools in Chlamydomonas reinhardtii chloroplasts, eLife 7:e37960
 
Alseekh S, Tong H, Scossa F, Brotman Y, Vigroux F, Tohge T, Ofner I, Zamir D, Nikoloski Z, Fernie AR (2017) Canalization of Tomato Fruit Metabolism, Plant Cell 29(11): 2753-2765.
 
Nikoloski Z., Balser G., Larhlimi A., Barabasi A.-L., Liu Y.-Y. (2016) Control of fluxes in metabolic networks, Genome Research 26(7): 956–968.
 
Omranian N., Kleessen S., Toghe T., Basler B., Mueller-Roeber B., Fernie A. R., Nikoloski, Z. (2015) Differential plant metabolic and coexpression networks, Trends in Plant Science 20(5): 266–268.
 
Kleessen S., Laitinen R., Fusari C., Antonio C., Sulpice R., Fernie A.R., Stitt M., Nikoloski Z.  (2014). Metabolic efficiency underpins performance trade-offs in growth of Arabidopsis thaliana. Nature Communications 5:3537.

## Seyfried
Otten, C., Knox, J., Boulday, G., Eymery, M., Haniszewski, M., Neuenschwander, M., Radetzki, S., Vogt, I., Hähn, K., De Luca, C., Cardoso, C., Hamad, S., Igual Gil, C., Roy, P., Albiges-Rizo, C., Faurobert, E., von Kries, J.P., Campillos, M., Tournier-Lasserve, E., Derry, W.B. & Abdelilah-Seyfried, S. Systematic pharmacological screens uncover novel pathways involved in cerebral cavernous malformations. EMBO Mol Med. pii: e9155 (2018).

Donat, S., Lourenco, M., Paolini, A., Otten, C., Renz, M. & Abdelilah-Seyfried, S. Heg1 and Ccm1/2 proteins control endocardial mechanosensitivity during zebrafish valvulogenesis. eLIFE 7, e28939 (2018).

Merks AM, Swinarski M, Meyer AM, Müller NV, Özcan I, Donat S, Burger A, Gilbert S, Mosimann C, Abdelilah-Seyfried S, Panáková D. Planar cell polarity signalling coordinates heart tube remodelling through tissue-scale polarisation of actomyosin activity. Nat Commun. 9, 2161 (2018).

Renz, M., Otten, C., Faurobert, E., Rudolph, F., Zhu, Y., Boulday, G., Duchene, J., Mickoleit, M., Dietrich, A.C., Ramspacher, C., Steed, E., Manet-Dupé, S., Benz, A., Hassel, D., Vermot, J., Huisken, J., Tournier-Lasserve, E., Felbor, U., Sure, U., Albiges-Rizo, C. & Abdelilah-Seyfried, S. Regulation of β1 Integrin-Klf2-mediated angiogenesis by CCM proteins. Dev. Cell 32, 181-190 (2015).

Dietrich, A.C., Lombardo, V., Veerkamp, J., Priller, F. & Abdelilah-Seyfried, S. 2014. Blood flow and Bmp signaling control endocardial chamber morphogenesis. Dev. Cell 30, 367-377 (2014).


## Tiedemann
Autenrieth M, Hartmann S, Lah L, Roos A, Dennis AB, Tiedemann R (2018) High quality whole genome sequence of an abundant Holarctic odontocete, the harbour porpoise (Phocoena phocoena). Molecular Ecology Resources 18, 1469-1481.

Schedina IM, Groth D, Schlupp I, Tiedemann R (2018) The gonadal transcriptome of the unisexual Amazon molly Poecilia formosa in comparison to its sexual ancestors, Poecilia mexicana and Poecilia latipinna. BMC Genomics 19, 12.

Valente V, Illera JC, Havenstein K, Pallien T, Etienne RS, Tiedemann R (2017) Equilibrium bird species diversity in Atlantic islands. Current Biology 27, 1660-1668.

Lamanna F, Kirschbaum F, Ernst ARR, Feulner PGD, Mamonekene V, Paul C, Tiedemann R (2016) Species delimitation and phylogenetic relationships in a genus of African weakly-electric fishes (Osteoglossiformes, Mormyridae, Campylomormyrus). Molecular Phylogenetics and Evolution 101, 8-18.

Schwarte S, Wegner F, Havenstein K, Groth D, Steup M, Tiedemann R (2015) Sequence variation, differential expression, and divergent evolution in starch-related genes among accessions of Arabidopsis thaliana. Plant Molecular Biology 87, 489-519.
